calaccess_raw/management/commands/__init__.py

```diff
@@ -20,6 +45,109 @@ def handle(self, *args, **options):
         self.verbosity = options.get("verbosity")
         self.no_color = options.get("no_color")
 
+        self.start_datetime = datetime.now()
+
+        # TODO: see if there's another way to identify caller
+        # in (edge) case when update is not called from command line
+        # if called from command line
+        if self._called_from_command_line:
+            # use the arg passed to manage.py as command name
+            self.command_name = sys.argv[1]
+        else:
+            # or take the end of the command's full module name
+            self.command_name = sub(r'(.+\.)*', '', self.__class__.__module__)
+
+        self.database = options["database"] or 'default'
```

Comment:
I think just leave `self.database = None` if it's not specified. The default is not `'default'`, but rather to allow Django to route the model. Release control :)